tagmap.dvc

```diff
@@ -1,10 +1,11 @@
 cmd: python3 tagmap/tagmap_gen.py
 deps:
```

Comment:
No `tagmap/tagmap_gen.py` in dependencies, I know this is not the only one but let's start fixing them as we make changes.